src/mono/mono/eventpipe/ep-rt-mono.c

```diff
@@ -2069,6 +2082,252 @@ ep_rt_mono_rand_try_get_bytes (
 	return mono_rand_try_get_bytes (&_ep_rt_mono_rand_provider, (guchar *)buffer, (gssize)buffer_size, error);
 }
 
+static GString *
+quote_escape_and_append_string (char *src_str, GString *target_str)
+{
+#ifdef HOST_WIN32
+	char quote_char = '\"';
+	char escape_chars[] = "\"\\";
+#else
+	char quote_char = '\'';
+	char escape_chars[] = "\'\\";
+#endif
+
+	gboolean need_quote = FALSE;
```

Comment:
I know we ported this code into EventPipe, but should we try to switch and use C99 types as much as possible in ported code. If we call function using Mono types, we should of course use matching types, but for other things, I think we should try to align with EventPipe code.